### PR TITLE
Benchmark updates

### PR DESCRIPTION
**Pull request by izdeby**
- Fixed input generation
- Updated padding mechanism
- Fixed cross entropy for NT

### CODE REVIEW

File: benchmarks/segmentation_layers.py [hunk 1]
```diff
@@ -94,10 +50,10 @@ def _relu_tensor_iter():
         return _relu_tensor_iter
 
     def relu_tensor_pad(self):
-        inputs = torch.stack(self.pad_tensors_to_max_shape(self.inputs))
+        tensor, mask = nestedtensor.nested_tensor(self.inputs).to_tensor_mask()
```
Comment (cpuhrsch) [inline benchmarks/segmentation_layers.py:53]: In python it is convention to use "_" for an unused variable. So you'd replace "mask" by "_" here. Some linters will even throw warnings if you don't.

Comment (izdeby) [reply benchmarks/segmentation_layers.py]: fixed
